src/routes/Edit/VideoSplitter.tsx

```diff
@@ -185,6 +210,17 @@ const VideoSplitter = ({
   };
 
   const saveChanges = async () => {
+    console.log(currentSegment?.title.length);
```

Comment:
Unless you believe logs will be helpful in production lets try not to merge them into master.

Reply:
Sure. I removed many. Thanks.